<ui version="4.0" >
 <class>FServerWindow</class>
 <widget class="QMainWindow" name="FServerWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RemoteControl - Serverlist</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <property name="horizontalSpacing" >
     <number>6</number>
    </property>
    <property name="verticalSpacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Servers</string>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>220</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>220</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>Server Info</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" colspan="3" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="4" >
           <widget class="QLineEdit" name="eVersion" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="4" >
           <widget class="QLineEdit" name="eLanguage" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="lLanguage" >
            <property name="text" >
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lVersion" >
            <property name="text" >
             <string>Version</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="6" >
           <widget class="QTextEdit" name="eDescription" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Website</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="4" >
           <widget class="QLineEdit" name="eHomepage" />
          </item>
          <item row="4" column="5" >
           <widget class="QPushButton" name="pWebsite" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>23</height>
             </size>
            </property>
            <property name="text" >
             <string>></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item rowspan="2" row="0" column="0" >
        <widget class="QTreeWidget" name="treeWidget" >
         <property name="lineWidth" >
          <number>1</number>
         </property>
         <property name="midLineWidth" >
          <number>0</number>
         </property>
         <property name="horizontalScrollBarPolicy" >
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="indentation" >
          <number>20</number>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights" >
          <bool>true</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pConnect" >
        <property name="text" >
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="pRefresh" >
        <property name="text" >
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>eLanguage</tabstop>
  <tabstop>eVersion</tabstop>
  <tabstop>eDescription</tabstop>
  <tabstop>pRefresh</tabstop>
  <tabstop>pConnect</tabstop>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc" />
 </resources>
 <connections/>
</ui>
